packages/data-context/src/data/coreDataShape.ts

```diff
@@ -127,6 +127,7 @@ interface CloudDataShape {
 
 export interface CoreDataShape {
   cliBrowser: string | null
+  isBrowserGivenByCLI: boolean
```

Comment:
Won't this just be `Boolean(cliBrowser)`?

Reply:
Nope

In cypress open mode, I think it would be the same

In cypress run mode, the default value of cliBrowser is ```electron``` so that's why a new variable is needed

Reply:
Right, I see.

Reply:
All functionality with that new variable works